reading/iex.livemd

```diff
@@ -275,6 +275,12 @@ helper to clear the console screen.
 iex> clear
 ```
 
+<!-- livebook:{"break_markdown":true} -->
+
+`iex(4)> clear`
```

Comment:
Good to know that's an issue on Windows.

I'm going to keep recommending we install WSL on your machine. More and more, I'm convinced that using Windows without WSL is going to be a problem.

Let me know when I eventually wear you down, and we can make some time to install it together ;)

Reply:
Thank you @BrooklinJazz . It is a case of climbing up Mount Everest. I run a very stripped down version of windows 10 called the LTSB edition, which does not have all the features of a regular edition of Windows 10. It is one of the recommended versions for media creators who need high performance machines without too many background services potentially interfering.

Long story short, my Windows 10 will not take WSL. I have freed enough space on my hard drive to create a dual boot and have a second operating system with Windows 11 and use it for coding. Unfortunately my SSD refuses to be defraged, hence I cannot create the partition for the dual boot big enough to solve the problem this way.

My next move is to bite the bullet and do a full re-install of my machine with Windows 11, but that is a multi-day operation and will interfere with running my production business as well as with a normal continuation of the student's schedule. It has to happen at some point, though. 

If you have any idea on how to go about this in an easy, sustainable way, I am all ears :)

Thanks!